addon/mixins/touch-action.js

```diff
@@ -45,7 +44,7 @@ export default Mixin.create({
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
     const hasTag = (typeof tagName === 'string' && tagName.length > 0) || (tagName === null && hasClick);
```

Comment:
oh yeah - meant to mention that tagName **can** be null and often, but not always is - which is why i check for `this.click.apply && tagName === null` in latest commit - seems normal nodes get parsed and have a tag name but curly's don't till much later (this is main change from glimmer1 I think or suspect) - just thought i'd clear that up - there was a change - i think there is still a bit of a life cycle bug/difference when mixing in - cool later